### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by jospoortvliet**
Add notes on the new template editing options.

### CODE REVIEW

**Comment by jospoortvliet** [comment]
Source: mostly via https://github.com/nextcloud/server/issues/8350#issuecomment-373837526 and https://github.com/nextcloud/server/issues/7229#issuecomment-364575856 ;-)

File: README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+}
+```
+
+3. Then in `config/config.php` change `mail_template_class` to your class namespace
+
+```php
+'mail_template_class' => MyVendor\MyClass::class
+```
+
+If you need any help, contact the Nextcloud support team [through the support portal.](https://portal.nextcloud.com)
```
Comment (MariusBluem) [inline README.md]: Directly link to https://portal.nextcloud.com/article/customized-email-templates-29.html

Comment (splitt3r) [reply README.md]: portal

Looks quite strange as a user who is not logged in 😛 btw.

Should look like:

portal

i think.

Comment (MariusBluem) [reply README.md]: Thanks for reporting.

Comment (MorrisJobke) [reply README.md]: Fixed.

**Review by MorrisJobke — APPROVED** [review]
Updated the link 👍

**Comment by MorrisJobke** [comment]
Just after merging I have noticed some typos, that I fixed in 295ceaa

**Pull request merged**